android/app/buildConfigs.gradle

```diff
@@ -1,7 +1,11 @@
 import groovy.json.JsonSlurper
 
 def isWindows() {
-    System.properties['os.name'].toLowerCase().contains('windows') ? "true" : "false"
+    System.properties['os.name'].toLowerCase().contains('windows')
+}
+
+def execCommand(command) {
+    isWindows() ? ["cmd", "/c", command] : command
```

Comment:
This function does not execute anything. Add the `.execute()` here.

Reply:
The lines afterwards can be added here too!